build/publish-WindowsAppSDKTelemetry.yml

```diff
@@ -0,0 +1,25 @@
+parameters:
```

Comment:
nit: Could you put this file in build/AzurePipelineTemplates and call it build/publish-WindowsAppSDKTelemetry-Steps.yml